src/api/table.ts

```diff
@@ -18,11 +18,14 @@ export const updateVenueTable = async ({
   subtitle,
   capacity,
 }: UpdateVenueTableProps) => {
+  const updatedCapacity = parseInt(capacity.toString(), 10);
```

Comment:
You are converting `capacity` to string and then you are parsing it back to number, why is that?

Reply:
the issue is that venue update type has the capacity listed as "string" type, when it should be a number
changing `capacity` type would require a lot of complementary changes throughout the application, because venue entity is used everywhere, thus I decided to implement this little hack

Reply:
can't we write something like this: 
```suggestion
  const updatedCapacity = parseInt(capacity, 10);
```
<hr />

if you both agree on having it as is, can you @alexey-petrov add a `@debt` comment here that would describe why it's needed so that we could fix that during the next code clean-up?

Reply:
I've added @debt comment there, so that we know the source of the issue
3aac5ee

Reply:
@alexey-petrov safest way of converting to string is the function `String()` (note, no `new` keyword in front of it). 
Something like `String( x ?? '' )` will not break with error if `Symbol` is provided and will make `null` and `undefined` empty strings ( or whichever default like `'10'` ) unlike the possibility of `null.toString()`, should data be 🤷 unexpected of type.

Either way, an update to the props is welcome, something like `capacity: number | string;` to closely reflect the reality of the issue